src/keyboard.c

```diff
@@ -95,12 +97,32 @@ static char keyboard_map(int code) {
             return KEY_INVALID;
         } else if (shiftlock_mode) {
             if (shift_mode) {
-                return keymap[code].normal;
+                if (keymap[code].normal >= 97 && keymap[code].normal <= 122) {
+                    return keymap[code].normal;
+                }
+                else {
```

Comment:
Style: all `else`'s should be on the same line as the closing curly brace of the `if` block